src/_includes/footer.hbs

```diff
@@ -0,0 +1,17 @@
+  <footer class="site-footer">
+      <div class="content">
+        <div class="col-5 hidden-xs">
+          <a href="https://mlh.io" class="mlh-logo white" alt="Major League Hacking" title="Major League Hacking"></a>
+        </div>
+        <div class="col-7 col-xs-12">
+          <p class="text-right text-center-xs">Major League Hacking, Inc &copy; 2017</p>
+        </div>
+      </div>
+    </footer>
+
+    {{> ./tracking/facebook}}
+    {{> ./tracking/twitter}}
+    {{> ./tracking/google_analytics}}
```

Comment:
We can actually replace all three of these with segment.  Segment automatically pushes to FB, Twitter, and GA.  https://segment.com/docs/sources/website/analytics.js/quickstart/